rita_common/src/usage_tracker/mod.rs

```diff
@@ -313,6 +422,102 @@ pub fn save_usage_on_shutdown() {
 
     let history = &mut USAGE_TRACKER.write().unwrap();
     history.last_save_hour = current_hour;
+    while history.payments.len() > MAX_TX_ENTRIES {
+        let _discarded_entry = history.payments.pop_back();
```

Comment:
why not use the trim_payments function here?

Reply:
fixed